Tests/StatKitTests/Descriptive Statistics Tests/AveragesTests.swift

```diff
@@ -11,6 +11,14 @@ final class AveragesTests: XCTestCase {
     
     XCTAssertEqual(calculatedMean, expectedMean)
   }
+  func testBigNumberIntegerArithmeticMean() {
+    let value = Int.max / 9
+    let bigIntArray = [9*value, 6*value, 3*value]
+    let calculateMean = bigIntArray.mean(of: \.self)
+    let expectedMean = Double(value*6)
+    print(expectedMean, calculateMean, bigIntArray)
+    XCTAssertEqual(calculateMean, expectedMean)
+  }
```

Comment:
The debug print should be taken out. :)

```suggestion

  func testBigNumberIntegerArithmeticMean() {
    let value = Int.max / 9
    let bigIntArray = [9 * value, 6 * value, 3 * value]
    let calculateMean = bigIntArray.mean(of: \.self)
    let expectedMean = Double(value * 6)
    
    XCTAssertEqual(calculateMean, expectedMean, accuracy: 0.00001)
  }
```